WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -738,15 +745,29 @@ class MySiteViewModel @Inject constructor(
     }
 
     fun onPullToRefresh() {
-        postCardsSource.refresh()
+        refresh()
```

Comment:
Suggestion (💡): Consider removing this extra `onPullToRefresh()` function from this view model and use directly the `refresh()`. This way, you wouldn't at least need to create another set of test for this publicly available function. Or is it that this is needed because in the future it will be doing more than just calling `refresh()`?

Reply:
Let's change it to refresh, but reserve the right to create a new fun if we need to differentiate between user requested PTR and automatic.

Reply:
in c67b923